Fix default settings to be sufficient to run app

vi4m opened the pull request:
RALPH_DEBUG=True will be env setting in deb default installation.
User can change it without writing config file. Default is False.

mkurek commented:
i can't agree with this change. can't you use dev settings in deb? Or create new file for deb settings?

vi4m commented:
It would require to copy all the file contents. I think getenv() variable will be better?

mkurek commented:
you could use (in `ralph.settings.deb` or sth similar (deb vs dev typo)):
```
from ralph.settings.prod import *
DEBUG = True
```
if you want to use env variable name it other than `DEBUG` (ex. `RALPH_FORCE_DEBUG`)

vi4m commented:
Why not RALPH_DEBUG ?

mkurek commented:
`RALPH_DEBUG` is ok too - it just has to be variable which is not-so-easy (™) to overwrite (user has to set this value on purpose, not accidentally)

vi4m committed:
Fix default settings to be sufficient to run app

RALPH_DEBUG=True will be env setting in deb default installation.
User can change it without writing config file.

vi4m commented:
Ok, this will come with RALPH_DEBUG env setting with the safe default value.

mkurek commented on src/ralph/settings/prod.py:
```diff
@@ -3,8 +3,7 @@
 
-# FIXME: when going for full production, change it to False
-DEBUG = False
+DEBUG = os.environ.get('RALPH_DEBUG', False) 
```
use `os_env_true` (it's shortcut to properly handle boolean env variables)

vi4m committed:
Review fixes + fixed missing docstring

vi4m committed:
Change os_env_true to bool_from_env

vi4m committed:
Added annotation

szok commented:
:ok:

mkurek commented:
:ok:
